src/atlantis/manager/helper/helper.go

```diff
@@ -192,7 +193,15 @@ func RegionAndZone(zone string) string {
 }
 
 func ZoneMinusRegion(zone string) string {
-	return strings.Replace(zone, Region, "", 1)
+	zmr := strings.Replace(zone, Region, "", 1)
+	if zmr == "" {
+		results := strings.Split(zone, "-")
+		if len(results) == 2 {
+			zmr = results[1]
+		}
+	}
+	log.Printf("ZoneMinusRegion: zone=%s Region=%s zmr='%s'", zone, Region, zmr)
```

Comment:
I'm not sure we want to be logging this regularly; I would expect that this function is called quite a bit, and this doesn't provide insight into how the services is working, aside from demonstrating that this function is correct.